### PR TITLE
Activities-add-jobs.html

### PR DESCRIPTION
**Pull request by enyia21**
#186714156

#### 🔗 Pivotal Tracker ticket
#186714156

#### ✍️ Description
1.  added `activities-add-jobs` screen
2. added the `activities-add-jobs-list` logic to the `activities-add-jobs` screen. see [pivotal](https://www.pivotaltracker.com/story/show/186714156/comments/240768131)
3.  added `activities-employer-name` screen
4. added `activities-employer-address` 
5. added `activities-self-employment`
6. added `delete-jobs` screen
7.  created a `jobs` subflow
added `message-properties` fields and tests

#### 📷 Design reference
[Figma activities-add-jobs](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=485-20111&mode=design&t=vAFS8hfgAq1Ak5tb-4)
[Figma activities-add-jobs-list](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=485-20186&mode=design&t=vAFS8hfgAq1Ak5tb-4)
[Figma activities-employer-name](https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=746-12950&mode=design&t=vAFS8hfgAq1Ak5tb-4)

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/resources/templates/gcc/activities-employer-name.html [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
\ No newline at end of file
+<th:block
+  th:replace="~{fragments/screens/screenWithOneInput ::
+  screenWithOneInput(
+    title=#{activities-employer-name.title},
+    header=#{activities-employer-name.header},
+    buttonLabel=#{general.button.continue},
+    subtext=#{activities-employer-name.subtext},
+    formAction=${formAction},
+    inputContent=~{::inputContent})}">
```
Comment (guiburato) [inline src/main/resources/templates/gcc/activities-employer-name.html:9]: The field is required.

https://www.figma.com/file/1oRFY7p6tdGogBJk7Ugle8/IL-CCAP-Full-Flow-(WIP)?type=design&node-id=3046-11083&mode=design&t=hDvEG7ArUGDTAxn0-0

**Comment by guiburato** [comment]
When you enter a name on "activities-employer-name" and click continue and click to "Go back," the field does not populate and always creates a "new Job".

File: src/main/resources/templates/gcc/activities-employer-name.html [hunk 2]
```diff
@@ -1,28 +1,16 @@
...
-    </div>
-  </section>
-</div>
-<th:block th:replace="~{fragments/footer :: footer}"/>
-</body>
-</html>
\ No newline at end of file
+<th:block
+  th:replace="~{fragments/screens/screenWithOneInput ::
+  screenWithOneInput(
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-employer-name.html:3]: Looks like this is missing the suitcase icon

File: src/main/resources/templates/gcc/activities-self-employment.html [hunk 3]
```diff
@@ -1,28 +1,13 @@
...
-</body>
-</html>
\ No newline at end of file
+<th:block
+  th:replace="~{fragments/screens/screenWithYesAndNoButtons ::
+  screenWithYesAndNoButtons(
+    title=#{activities-self-employment.title},
+    header=#{activities-self-employment.header},
+    subtext=#{activities-self-employment.subtext},
+    iconFragment=~{fragments/gcc-icons :: baby},
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-self-employment.html:7]: This page doesnt appear to have an icon
```suggestion
```

File: src/main/resources/templates/gcc/activities-self-employment.html [hunk 4]
```diff
@@ -1,28 +1,13 @@
...
+    formAction=${formAction},
+    inputName='isSelfEmployed',
+    ariaLabel='header',
+    inputContent=~{::inputContent})}">
+  <th:block th:ref="inputContent"/>
```
Comment (sree-cfa) [inline src/main/resources/templates/gcc/activities-self-employment.html:12]: We want the company name header here as well
```suggestion
  <th:block th:ref="inputContent">
    <th:block th:replace="~{fragments/inputs/text ::
        text(inputName='companyName',
        ariaLabel='header')}"/>
    </th:block>
</th:block>
```

**Review by sree-cfa — APPROVED** [review]
lgtm

**Pull request merged**